samples/java_springboot/21.corebot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/CloudAdapterWithErrorHandler.java

```diff
@@ -70,33 +67,33 @@ public AdapterWithErrorHandler(
             // NOTE: In production environment, you should consider logging this to
             // Azure Application Insights. Visit https://aka.ms/bottelemetry to see how
             // to add telemetry capture to your bot.
-            LoggerFactory.getLogger(AdapterWithErrorHandler.class).error("onTurnError", exception);
+            LoggerFactory.getLogger(CloudAdapterWithErrorHandler.class).error("onTurnError", exception);
 
             // Send a message to the user
             return turnContext.sendActivities(
-                MessageFactory.text(ERROR_MSG_ONE), MessageFactory.text(ERROR_MSG_TWO)
+                    MessageFactory.text(ERROR_MSG_ONE), MessageFactory.text(ERROR_MSG_TWO)
             ).thenCompose(resourceResponse -> sendTraceActivity(turnContext, exception))
-                .thenCompose(stageResult -> {
-                    if (withConversationState != null) {
-                        // Delete the conversationState for the current conversation to prevent the
-                        // bot from getting stuck in a error-loop caused by being in a bad state.
-                        // ConversationState should be thought of as similar to "cookie-state" in a
-                        // Web pages.
-                        return withConversationState.delete(turnContext)
-                            .exceptionally(deleteException -> {
-                                LoggerFactory.getLogger(AdapterWithErrorHandler.class)
-                                    .error("ConversationState.delete", deleteException);
-                                return null;
-                            });
-                    }
-                    return CompletableFuture.completedFuture(null);
-                });
+                    .thenCompose(stageResult -> {
+                        if (withConversationState != null) {
+                            // Delete the conversationState for the current conversation to prevent the
+                            // bot from getting stuck in a error-loop caused by being in a bad state.
+                            // ConversationState should be thought of as similar to "cookie-state" in a
+                            // Web pages.
+                            return withConversationState.delete(turnContext)
+                                    .exceptionally(deleteException -> {
+                                        LoggerFactory.getLogger(CloudAdapterWithErrorHandler.class)
+                                                .error("ConversationState.delete", deleteException);
+                                        return null;
+                                    });
+                        }
+                        return CompletableFuture.completedFuture(null);
+                    });
```

Comment:
I think the original source was using a indentation of 4 spaces while this code has 8 spaces. Can you fix it to make it consistent so the diff shows only actual changes?